core/src/main/java/com/graphhopper/routing/ev/InSubnetwork.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.routing.ev;
+
+public class InSubnetwork {
+    public static String key(String prefix) {
+        if (prefix.contains("subnetwork"))
```

Comment:
have now named it `subnetwork`. Not sure about the subclassing. You mean this would be safer?

Reply:
I just meant it might be easier to search and find usages or code related to `Subnetwork` if it was a separate class (dedicated subclass of SimpleBooleanEncodedValue)

Reply:
Currently all EncodedValue implementations (except UnsignedIntEncodedValue) are final  and I'm unsure if this is a valid reason to open the class. Because even if we use SubnetworkEncodedValue as separate class we won't create lookup.getSubnetworkEncodedValue and so adding a BooleanEncoedValue instead of a SubnetworkEncodedValue will still be valid. Additionally we should be able to find all usages via search for `Subnetwork.key`.

Reply:
Ok then let's leave it as it is.